bpf-rust-programs/prediction-poll/solana_types/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+#![no_std]
```

Comment:
Why not using the ones from rust-utils in the sdk?

Reply:
I thought that depending on utils would cause issues in the wasm crate. I just tested though and it's fine, I'll remove this crate